src/hash.cr

```diff
@@ -1752,8 +1752,12 @@ class Hash(K, V)
     self
   end
 
-  # Compares with *other*. Returns `true` if all key-value pairs are the same.
   def ==(other : Hash)
+    false
```

Comment:
This doesn't seem correct.

Reply:
Indeed, I called it out in the original PR description.